src/theme/ItaliaTheme/Blocks/_ribbonCardTemplate.scss

```diff
@@ -8,11 +8,11 @@
     .etichetta {
       position: absolute;
       top: 3rem;
-      right: 0;
-      left: 0;
-      width: 100%;
+      right: 0px !important;
```

Comment:
Un important in questo tema che è la base per tutti dev'essere molto ben giustificato. Non c'è modo di non usarlo?